src/compile/front.rs

```diff
@@ -113,6 +113,8 @@ async fn bindgen(proj: &Project) -> Result<Outcome<Product>> {
     // https://github.com/rustwasm/wasm-bindgen/blob/main/crates/cli-support/src/lib.rs#L95
     // https://github.com/rustwasm/wasm-bindgen/blob/main/crates/cli/src/bin/wasm-bindgen.rs#L13
     let mut bindgen = Bindgen::new()
+        .debug(proj.wasm_debug)
+        .keep_debug(proj.wasm_debug)
```

Comment:
If we wanted we could split these up into two flags, but I think most people will want to enable both. If you enable keep_debug but not debug you get file names in stack traces but clicking them no longer opens the Rust source files.